### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by zapion**
Update README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,8 +6,20 @@ Let's see a example from template.json
 
 {
   "name": "b2g-info cmd",
+  "type": "",
```
Comment (ShakoHo) [inline README.md]: suggest we can give a sample here.
for example: "type": "moz_minions.minions.ShellMinion",

Comment (zapion) [inline README.md]: I am thinking of giving a default value here.  Thoughts?

Comment (ShakoHo) [inline README.md]: Ya, that's ok. If we give default value, we have to output more default setting value at first beginning of running the boss.

**Comment by ShakoHo** [comment]
lgtm r+

**Pull request merged**